vocabularies/seismic-dimensionality.ttl

```diff
@@ -1,19 +1,53 @@
- @prefix : <http://linked.data.gov.au/def/seismic-sampling-method> .
- @prefix ssmethod: <> .
+@prefix : <http://linked.data.gov.au/def/seismic-dimensionality> .
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sdo: <https://schema.org/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix ssmethod: <file:///tmp/skosify-cgiMwa8Mb/input.ttl> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
- @prefix dc: <http://purl.org/dc/elements/1.1/> .
- @prefix dct: <http://purl.org/dc/terms/> .
- @prefix owl: <http://www.w3.org/2002/07/owl#> .
- @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
- @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
- @prefix sdo: <https://schema.org/> .
- @prefix skos: <http://www.w3.org/2004/02/skos/core#> .
- @prefix xml: <http://www.w3.org/XML/1998/namespace> .
- @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+<http://linked.data.gov.au/def/seismic-dimensionality> a skos:ConceptScheme ;

```

Comment:
:conceptScheme a skos:ConceptScheme
    dct:created "2019-09-24"^^xsd:date ;
    dct:modified "2019-09-24"^^xsd:date ;
    dct:creator "Geological Survey of Queensland" ;

change concept scheme lines and add temporal lines (change dates if known/needed)